dotcom-rendering/scripts/webpack/dev/setup-middlewares.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const express = require('express');
+const webpackHotServerMiddleware = require('webpack-hot-server-middleware');
+const bodyParser = require('body-parser');
+const {
+	getContentFromURLMiddleware,
+} = require('../../../src/server/lib/get-content-from-url');
+
+module.exports = (middlewares, devServer) => {
```

Comment:
If you want a little bit of type info, you can do this:

```suggestion
/**
 * @param {Array<import('webpack-dev-server').Middleware>} middlewares
 * @param {import('webpack-dev-server')} devServer
 * @returns {Array<import('webpack-dev-server').Middleware>}
 */
module.exports = (middlewares, devServer) => {
```